dev/com.ibm.ws.request.probes/src/com/ibm/wsspi/requestContext/RequestContext.java

```diff
@@ -38,6 +38,7 @@
 	private volatile int state = -1;
 	private int eventCount = -1;
 	private static final RequestIdGeneratorPUID idgen = new RequestIdGeneratorPUID();
+	public boolean isSlow;
```

Comment:
Should `isSlow` be private?

Reply:
Please also update copyright years to 2014, 2019

Reply:
isSlow has been changed to private and the copyright years has been updated

Reply:
Changed isSlow to private and updated the copyright years.